content/docs/command-reference/plots/templates.md

```diff
@@ -0,0 +1,97 @@
+## plot templates
+
+Dump plot templates.
+
+Sometimes users need to customize the way project data is visualized. This
+command allows dumping templates used by DVC, so that the users don't have to
+build their own templates from zero.
+
+## Synopsis
+
+```usage
+usage: dvc plots templates [-h] [-q | -v] [-o <path>] [{simple,linear,confusion,confusion_normalized,scatter,smooth}]
+
+positional arguments:
+  TEMPLATE
```

Comment:
I think we are using lower case names?

Reply:
Moved to https://github.com/iterative/dvc.org/pull/3697#pullrequestreview-1024032767